returnn/frontend/linear.py

```diff
@@ -40,6 +41,43 @@ def __call__(self, source: Tensor) -> Tensor:
         return out
 
 
+class LinearWithWeightDropout(rf.Module):
```

Comment:
Why do you add this? We don't want to have specialized variants for every single existing module for every single variant of reparameterization (e.g. weight dropout, variational noise, weight normalization, ...). See #1518.